ha/util/conf_store.py

```diff
@@ -221,3 +221,30 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_pods(index):
+        """
+        Returns machine_id for control-node pod.
+        Args:
+            index(str): index of conf file
+            parent key : node
+            search key: type
+            search val: control_node
+        Returns:
+            list: list of node_ids
+
+        >>> Conf.search("cortx", 'node', 'type', 'control_node')
+        ['node>a3710eee36644f2fb60ba19486664495>type']
+        """
+        node_id = []
```

Comment:
suggestion: to change this to node_ids, node_id_list, or node_list

Reply:
ACK